modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceSearch.testcase

```diff
@@ -65,6 +65,116 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-9998 We can see all order results on the Global Search by email"
+	@priority = "3"
+	test CanSeeAllOrdersOnTheGlobalSearchByEmail {
+		property portal.acceptance = "false";
+
+		task ("Given a Minium site with a buyer user") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+
+			CommerceEntry.addAccountEntryUser(
+				accountName = "Commerce Account",
+				accountType = "Business",
+				agreeToTermsAndAnswerReminderQuery = "true",
+				createNewAccount = "true",
+				requireReset = "false",
+				userEmailAddress = "buyer@liferay.com",
+				userFirstName = "Buyer",
+				userLastName = "User",
+				userRole = "Buyer",
+				userScreenName = "buyeruser",
+				userSiteMembership = "Minium");
+		}
+
+		task ("And an order is placed by the buyer user") {
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "buyer@liferay.com");
+
+			FrontStore.addInitializerProductToCart(
+				productName = "U-Joint",
+				productQuantity = "1");
+
+			CommerceCheckout.gotoCheckout();
+
+			CommerceCheckout.initializersCheckout(
+				newAddressAddress = "Test Address",
+				newAddressCity = "Test City",
+				newAddressCountry = "United States",
+				newAddressName = "Address Name",
+				newAddressZipCode = "Test Zip Code",
+				productName = "U-Joint",
+				productQuantity = "1",
+				productUnitPrice = "24",
+				shippingMethod = "Standard");
+		}
```

Comment:
@baharturk this can be done using API calls

Reply:
The aim of this test is to do use global search by passing the email address of the buyer user, but the macro used for placing an order with an API call does not support doing it that way so I'm keeping the test as it is now, we also discussed it with the reviewer.